lerobot/common/robot_devices/control_utils.py

```diff
@@ -246,14 +263,21 @@ def control_loop(
     while timestamp < control_time_s:
         start_loop_t = time.perf_counter()
 
+        current_joint_positions = robot.follower_arms["main"].read("Present_Position")
```

Comment:
A small fix, linter generate an error

<img width="1071" alt="Screenshot 2025-03-29 at 1 25 25 AM" src="https://github.com/user-attachments/assets/2c2892ba-d6d8-4179-96c9-5ddfbe9941e3" />
```suggestion
        # current_joint_positions = robot.follower_arms["main"].read("Present_Position")
```